### ISSUE TITLE
Kind e2e test failure when pulling flow-aggregator image

### ISSUE THREAD

**Issue by yanjunz97** [antrea-io/theia#67]
**Describe the bug**
Recently I have noticed the kind e2e test sometimes failed with the errors below. The error may not show up after re-running the test, but I'm not sure what is the root cause. 
```
 Events:
          Type     Reason     Age                    From               Message
          ----     ------     ----                   ----               -------
          Normal   Scheduled  6m13s                  default-scheduler  Successfully assigned flow-aggregator/flow-aggregator-95f95b7bd-4lxj6 to kind-worker2
          Normal   Pulling    4m37s (x4 over 6m13s)  kubelet            Pulling image "projects.registry.vmware.com/antrea/flow-aggregator:latest"
          Warning  Failed     4m37s (x4 over 6m12s)  kubelet            Failed to pull image "projects.registry.vmware.com/antrea/flow-aggregator:latest": rpc error: code = NotFound desc = failed to pull and unpack image "projects.registry.vmware.com/antrea/flow-aggregator:latest": failed to copy: httpReadSeeker: failed open: content at https://projects.registry.vmware.com/v2/antrea/flow-aggregator/manifests/sha256:b3594bd6fa1a8c5b12f2bdda5a4ddc5f7903f892660a1b4f832b245cf6082f77 not found: not found
          Warning  Failed     4m37s (x4 over 6m12s)  kubelet            Error: ErrImagePull
          Warning  Failed     4m23s (x6 over 6m12s)  kubelet            Error: ImagePullBackOff
          Normal   BackOff    67s (x20 over 6m12s)   kubelet            Back-off pulling image "projects.registry.vmware.com/antrea/flow-aggregator:latest"
```
Here are some cases for this failure:

- https://github.com/antrea-io/theia/runs/7104761249?check_suite_focus=true
- https://github.com/antrea-io/theia/runs/7100491020?check_suite_focus=true
- https://github.com/antrea-io/theia/runs/7099069286?check_suite_focus=true
- https://github.com/antrea-io/theia/runs/7048586961?check_suite_focus=true

**Comment by heanlan** [comment]
It is probably caused by the habor registry cache issue. Antrea recently merged a PR https://github.com/antrea-io/antrea/pull/3642 which updates the flow-aggregator image. I added a commit in PR #71 to replace the harbor registry image to dockerhub image.

**Comment by heanlan** [comment]
Closed via #71
